frontend/src/_components/CreateApp.jsx

```diff
@@ -0,0 +1,84 @@
+import React, { useState, useEffect, useContext } from 'react';
+import { toast } from 'react-hot-toast';
+import Modal from '../HomePage/Modal';
+import { ButtonSolid } from '@/_ui/AppButton/AppButton';
+import { appService } from '@/_services';
+import _ from 'lodash';
```

Comment:
You should directly import the method that you are using instead of importing the whole Lodash library.

It will improve tree-shaking